doc/api/events.md

```diff
@@ -474,6 +496,8 @@ finishes execution will not remove them from `emit()` in progress. Subsequent
 events will behave as expected.
 
 ```js
+'use strict';
+
```

Comment:
This example both works without `use strict` and is partial (a continuation of the first example), that means that this is not the correct place for `use strict`.